### ISSUE TITLE
Add a dummy background class to the front when using sigmoid

### ISSUE THREAD

**Issue by ArseniuML** [tusen-ai/SST#200]
In file fsdv2_head.py:
```

# Add a dummy background class to the front when using sigmoid
padding = scores.new_zeros(scores.shape[0], 1)
scores = torch.cat([scores, padding], dim=1)

score_thr = cfg.get('score_thr', 0)
results = box3d_multiclass_nms(bboxes, bboxes_for_nms,
                            scores, score_thr, cfg.max_num,
                            cfg)
```
                            
And box3d_multiclass_nms:

```
num_classes = mlvl_scores.shape[1] - 1
bboxes = []
scores = []
labels = []
dir_scores = []
attr_scores = []
bboxes2d = []
for i in range(0, num_classes):
....
```

So It seems that padding of scores does nothing...

1. What was the idea of this padding?
2. Why current implementation effectively does nothing?
